core/sql/src/main/java/org/trafodion/sql/udr/LmUDRObjMethodInvoke.java

```diff
@@ -68,7 +68,7 @@ static LmUDRObjMethodInvoke makeNewObj(
         return result;
     }
 
-    public static class ReturnInfo
+    public class ReturnInfo
```

Comment:
Yes, what you say in your last sentence is what I'm thinking as well. If a nested class doesn't need an association with an object of its parent class then it is preferable to make it static. In this case, I feel that passing a ReturnInfo object across JNI is a bit strange with an inner class, as we would separate the two related objects.